src/mdacli/cli.py

```diff
@@ -7,10 +7,15 @@
 # SPDX-License-Identifier: GPL-2.0-or-later
 
 """
-Main entry point for the MDAnalysis CLI interface.
+A command line interface (CLI) to the analysis modules of MDAnalysis.
 
-This also demonstrates how other third party libraries could incorporate
-this functionality.
+The modules are all structured as part of a single mdacli wrapper, and invoked
+with commands like `mdacli RMSD`. Documentation for each module can be found at
+the respective sections on the `MDAnalysis Analysis Documentation`_, as well as
+`mdacli command -h`.
+
+
+.. _`MDAnalysis Documentation`: https://docs.mdanalysis.org/stable/documentation_pages/analysis_modules.html
```

Comment:
reST allows you to put the URL on the next line.
```suggestion
.. _`MDAnalysis Documentation`: 
   https://docs.mdanalysis.org/stable/documentation_pages/analysis_modules.html
```